ex04.ex

```diff
@@ -55,7 +55,14 @@ defmodule Ex04 do
 
   """
 
-  def min . . . "your code"
+  def min([ h | t ]), do: reduce([ h | t ],
```

Comment:
## -2

Two things:

First, when the function gets longer than a few lines, see if it can be split.

Two, I really wanted you to avoid `if`, `cond`, and `case`.

~~~ elixir
  def min(list), do: reduce(list, &min_of/2)
  def min_of(a,  b) when a < b, do: a
  def min_of(_a, b), do: b
~~~